README.md

```diff
@@ -0,0 +1,109 @@
+
+Branch protection: 
+- PM's and QA can merge or commit directly to the master branch
+- PR's do not require approval to merge
+
+
+## Creating a notice
+
+### Before opening a PR
+
+1. Check how many notices are live in production in `notices.json`: As a general guideline we should aim to have < 5 live notices that could apply to any unique user to avoid spamming. Please reach out to the PM owner for any live notices if we are at the threshold and need to consider removing or replacing a notice. 
+2. Consider your audience: Carefully consider who should see this notice. Do we need to spam all users or should we target System Admins only? 
+3. Consider narrowing your reach: Consider narrowing your target audience further by using additional conditions such as SKU, client, client versions, server versions, server configs 
+
+### Opening a PR 
+Open a PR against the `release` branch to add your notice to the `notices.json`. See this [example PR] XXXXXX will submit a PR for the v5.29 server upgrade
+
+- **"id"**: must be a **unique** alpha-numeric value. The server uses the notice ID to store if a user has seen a particular notice. It is recommended to choose an ID that will be recognizable in telemetry.
+- **"title"**: configure the title of the modal and appears in larger bold font (supports emoji)
+- **"description"**: configure the description text that appears below the title (supports full markdown and emoji)
+- **"image"** (optional): configure the hero image that appears below the description text. It is recommended to use a 16:9 aspect ratio image ([example](https://github.com/mattermost/notices/blob/master/images/desktop_upgrade.png)). Images should be added to the `images` folder in your PR branch, then in the `notices.json` add the image URL in the format `https://raw.githubusercontent.com/mattermost/notices/release/images/image_name.image_format` ([example](https://github.com/mattermost/notices/pull/49/files#diff-11766faeb8c25f77d7dbf8e61fd0e9fc8cd1a08858d6b1f8867715a570bfd9d9R13))  
+- **"actionText"** (optional): configure the text in the call-to-action button for this notice. If nothing is specified this reverts to "Done". 
+- **"actionParam"** (optional): configure the action that the call-to-action button takes. This can be an external URL or system console page.  
+
+### After opening a PR
+1. Please add the "PM Review" and "QA Review" labels. Recommended reviewers are @esethna (PM) and @ogi.m or @jgilliam17 (QA). 
+2. QA will test the notice on the `master` branch and verify the conditions are working as expected then approve the PR. Once reviews are complete the PR can be merged to the `release` branch.
+3. Once merged, verify the changes have been mirrored to https://notices.mattermost.com/ (this may take up to 15 minutes post-merge).
+
+Servers check for notices https://notices.mattermost.com/ once per day so it will take up to 24 hours for any newly merged notices to appear in the wild.
```

Comment:
Notices are checked once an hour by default, not 24 hours

Reply:
Updated, thx